Code/src/main/java/nl/utwente/group10/ui/components/FunctionBlock.java

```diff
@@ -64,8 +57,7 @@ public String executeMethod() {
 	 * @param node to nest
 	 */
 	public void nest(Node node) {
-		Pane nestSpace = (Pane) this.lookup("#nest_space");
-		((Label) this.lookup("#label_function_name")).setText("Higher order function");
+		name.set("Higher order function");
```

Comment:
This will set the name property to the given string.

Behind the scenes, the actual visible label is bound to (set to observe) this property. It will automatically update itself to show the new value, if needed.